aerospike-mixin/alerts/alerts.libsonnet

```
@@ -0,0 +1,154 @@
+                  'which is above the threshold of %(alertsCriticalNodeHighMemoryUsage)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'AerospikeNamespaceHighDiskUsage',
+            expr: |||
+              100 - sum without (service) (aerospike_namespace_device_free_pct) >= %(alertsCriticalNamespaceHighDiskUsage)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'There is a limited amount of disk space available for a node.',
+              description:
+                (
+                  '{{ printf "%%.0f" $value }} percent of disk space available for namespace {{$labels.ns}} on node {{$labels.instance}}, on cluster {{$labels.aerospike_cluster}}, ' +
+                  'which is above the threshold of %(alertsCriticalNamespaceHighDiskUsage)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'AerospikeUnavailablePartitions',
+            expr: |||
+              sum without(service) (aerospike_namespace_unavailable_partitions) > %(alertsCriticalUnavailablePartitions)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'There are unavailable partitions in the Aerospike cluster.',
+              description:
+                (
+                  '{{ printf "%%.0f" $value }} unavailable partition(s) in namespace {{$labels.ns}}, on node {{$labels.instance}}, on cluster {{$labels.aerospike_cluster}}, ' +
+                  'which is above the threshold of %(alertsCriticalUnavailablePartitions)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'AerospikeDeadPartitions',
+            expr: |||
+              sum without(service) (aerospike_namespace_dead_partitions) > %(alertsCriticalDeadPartitions)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              summary: 'There are dead partitions in the Aerospike cluster.',
+              description:
+                (
+                  '{{ printf "%%.0f" $value }} dead partition(s) in namespace {{$labels.ns}}, on node {{$labels.instance}}, on cluster {{$labels.aerospike_cluster}}, ' +
+                  'which is above the threshold of %(alertsCriticalDeadPartitions)s.'
+                ) % $._config,
+            },
+          },
+          {
+            alert: 'AerospikeNamespaceRejectingWrites',
+            expr: |||
+              sum without(service) (aerospike_namespace_stop_writes + aerospike_namespace_clock_skew_stop_writes) > %(alertsCriticalSystemRejectingWrites)s
+            ||| % $._config,
+            'for': '1m',
```

```suggestion
            'for': '5m',
```
Consider to increase to 5m. Basically, In 1 minute, in Grafana Cloud only single point would be collected. But we need to get some confirmation that problem is there before bothering people with paging.